src/mewbot/demo.py

```diff
@@ -2,6 +2,12 @@
 #
 # SPDX-License-Identifier: BSD-2-Clause
 
+"""
+Simple example elements which demonstrate some of the most basic mewbot objects.
```

Comment:
Currently the basic elements here are used in a few places in the yaml example files.

I think there's some value to keeping some basic elements - stuff useful for debug - in the main code and not in the examples (which we might end up not shipping in all cases). Suggest shifting them to something like mewbot.utils.common_elements?

Reply:
I suspect there's going to have to be an API module for Event protocols (e.g. `Replyable`) and non specific conditions and actions so it might be worth taking the actual components from this module over there, and the rest out to examples?

Reply:
Makes sense to me! Do you want to part it down? (as I'm not sure where you'd want it to go in the API module).

If you do, I can take a look this evening.

Reply:
Moved the thought to https://github.com/mewler/mewbot/issues/162 as it's not a part of this change and isn't pressing.